### DISCUSSION TITLE
NI 63xx card as a fast counter?

### DISCUSSION THREAD

**Discussion by gadamc** [Ulm-IQO/qudi-core#22] [Q&A] [answered]
Hi all.
Has anybody written a FastCounterInterface for NI 63xx cards? We'd like to use that for pulsed ODMR. 
Thank you!

**Comment by Neverhorst** [comment, marked as answer]
Hi Adam,
not to my knowledge, no.
But you may have a good starting point for an implementation if you have a look at [this module](https://github.com/Ulm-IQO/qudi-iqo-modules/blob/main/src/qudi/hardware/ni_x_series/ni_x_series_finite_sampling_input.py).
It is the wrong interface (`FiniteSamplingInputInterface`) but implements ~90% of what you need.
Cheers!

**Reply by gadamc** [reply]
Thanks @Neverhorst I'll consider that.
